src/coreclr/scripts/jitrollingbuild.py

```diff
@@ -172,6 +182,111 @@ def determine_jit_name(coreclr_args):
+        3. Figure out the latest hash, starting with `base_git_hash`, that contains any changes to
+           the src/coreclr/jit directory. (We do this because the JIT rolling build only includes
+           builds for changes to this directory. So, this logic needs to stay in sync with the logic
+           that determines what causes the JIT rolling build to run. E.g., it should also get
+           rebuilt if the JIT-EE interface GUID changes. Alternatively, we can take the entire list
+           of changes, and probe the rolling build drop for all of them.)
+        4. Starting with `base_git_hash`, and possibly walking to older changes, look for matching builds
+           in the JIT rolling build drops.
+        5. If a JIT directory in Azure Storage is found, set coreclr_args.git_hash to that git hash to use
+           for downloading.
+
+    Args:
+        coreclr_args (CoreclrArguments) : parsed args
+
+    Returns:
+        Nothing
+
+        coreclr_args.git_hash is set to the git hash to use
+
+        An exception is thrown if the `-git_hash` argument is unspecified, and we don't find an appropriate
+        JIT to download.
+    """
+
+    if coreclr_args.git_hash is not None:
+        return
+
+    # Do all the remaining commands, including a number of 'git' commands including relative paths,
+    # from the root of the runtime repo.
+
+    with ChangeDir(coreclr_args.runtime_repo_location):
+        command = [ "git", "rev-parse", "HEAD" ]
+        print("Invoking: {}".format(" ".join(command)))
+        proc = subprocess.Popen(command, stdout=subprocess.PIPE)
+        stdout_git_rev_parse, _ = proc.communicate()
+        return_code = proc.returncode
+        if return_code == 0:
+            current_git_hash = stdout_git_rev_parse.decode('utf-8').strip()
+            print("Current hash: {}".format(current_git_hash))
+        else:
+            raise RuntimeError("Couldn't determine current git hash")
+
+        # We've got the current hash; figure out the baseline hash.
+        command = [ "git", "merge-base", current_git_hash, "origin/main" ]
+        print("Invoking: {}".format(" ".join(command)))
+        proc = subprocess.Popen(command, stdout=subprocess.PIPE)
+        stdout_git_merge_base, _ = proc.communicate()
+        return_code = proc.returncode
+        if return_code == 0:
+            base_git_hash = stdout_git_merge_base.decode('utf-8').strip()
+            print("Baseline hash: {}".format(base_git_hash))
+        else:
+            raise RuntimeError("Couldn't determine baseline git hash")
+
+        # Enumerate the last 20 changes, starting with the baseline, that included JIT changes.
+        command = [ "git", "log", "--pretty=format:%H", base_git_hash, "-20", "--", "src/coreclr/jit/*" ]
+        print("Invoking: {}".format(" ".join(command)))
+        proc = subprocess.Popen(command, stdout=subprocess.PIPE)
+        stdout_change_list, _ = proc.communicate()
```

Comment:
Worth printing the output of `stdout_change_list` for ease debugging.

Reply:
We do print each hash that we look at in the returned output. Maybe that's sufficient in the normal case?